powder_reduction.py

```diff
@@ -208,7 +215,18 @@ def powder_reduction(sample='sample.nxs',
 
         vana_red_focused.coords['detector_info'] = focused.coords['detector_info'].copy()
 
-        focused /= vana_red_focused
+        # realign vanadium and sample data in d-space
```

Comment:
If we use `groupby` on the realigned data (see previous comment), is this step necessary? The realignment should in principle carry through all operations (if it does not this is something we need to fix), it should only have to be done once.

Reply:
See above comment